app/helpers/application_helper/toolbar/basic.rb

```diff
@@ -44,9 +42,43 @@ def separator
     {:separator => true}
   end
 
+  def definition
+    unless @loaded
+      load_overrides
+      @loaded = true
+    end
+
+    @definition
```

Comment:
Prefer something like `@definition ||= load_overrides` and then have load_overrides just return the thing.

Reply:
Ugh...I see that `custom_content` injects itself via the ivars...might be better if those methods all go through the definition accessor.  That way they are loaded "first".  I guess I don't understand the DSL in this class and what is trying to be solved, so perhaps my suggestions should be ignored 😁

Reply:
I will be redoing that all. So while your comments make sense, I'll just dump the code and do it differently once the autoloading works.